src/Mvc/Mvc.Core/src/ControllerBase.cs

```diff
@@ -1849,6 +1898,8 @@ public virtual ActionResult ValidationProblem([ActionResultObjectValue] ModelSta
             }
 
             var validationProblem = new ValidationProblemDetails(modelStateDictionary);
+            ApplyProblemDetailsDefaults(validationProblem, statusCode: 400);
```

Comment:
Is this broken today? We don't apply the defaults?

Reply:
Yup